metricbeat/beater/module_test.go

```diff
@@ -0,0 +1,95 @@
+// +build !integration
+
+package beater_test
```

Comment:
I assume it is intentional that you use a test package here to have it tested from the outside. Perhaps worth a note?